cdap-proto/src/main/java/co/cask/cdap/proto/audit/AuditMessage.java

```diff
@@ -26,6 +26,8 @@
  */
 @Beta
 public class AuditMessage {
+  private final int version = 1;
```

Comment:
@anwar6953 downside is that by default, static variables are not serialized by GSON.